### ISSUE TITLE
"Go: Install/Update Tools" does not include dlv

### ISSUE THREAD

**Issue by joergjo** [golang/vscode-go#2339]
### What version of Go, VS Code & VS Code Go extension are you using?
<details><summary>Version Information</summary><br>

* Run `go version` to get version of Go from _the VS Code integrated terminal_.
	- go1.18.4 darwin/arm64
* Run `gopls -v version` to get version of Gopls from _the VS Code integrated terminal_.
	- golang.org/x/tools/gopls v0.9.1
* Run `code -v` or `code-insiders -v` to get version of VS Code or VS Code Insiders.
	- 1.69.1
* Check your installed extensions to get the version of the VS Code Go extension
	- v0.34.1
* Run Ctrl+Shift+P (Cmd+Shift+P on Mac OS) > `Go: Locate Configured Go Tools` command.

```
Checking configured tools....
GOBIN: undefined
toolsGopath: 
gopath: /Users/joergjo/go
GOROOT: /usr/local/go
PATH: /usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/go/bin:/Library/Apple/usr/bin:/Library/Frameworks/Mono.framework/Versions/Current/Commands:/Users/joergjo/.sdkman/candidates/springboot/current/bin:/Users/joergjo/.sdkman/candidates/quarkus/current/bin:/Users/joergjo/.sdkman/candidates/maven/current/bin:/Users/joergjo/.sdkman/candidates/java/current/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/Users/joergjo/go/bin:/opt/homebrew/opt/node@16/bin:/Users/joergjo/.dapr/bin:/Users/joergjo/.dotnet:/Users/joergjo/.dotnet/tools

	go:	/usr/local/go/bin/go: go version go1.18.4 darwin/arm64

	gotests:	/Users/joergjo/go/bin/gotests	(version: v1.6.0 built with go: go1.18.4)
	gomodifytags:	/Users/joergjo/go/bin/gomodifytags	(version: v1.16.0 built with go: go1.18.4)
	impl:	/Users/joergjo/go/bin/impl	(version: v1.1.0 built with go: go1.18.4)
	goplay:	/Users/joergjo/go/bin/goplay	(version: v1.0.0 built with go: go1.18.4)
	staticcheck:	/Users/joergjo/go/bin/staticcheck	(version: v0.3.2 built with go: go1.18.4)
	gopls:	/Users/joergjo/go/bin/gopls	(version: v0.9.1 built with go: go1.18.4)
[snip]
```
</details>

### Share the Go related settings you have added/edited

```json
"go.toolsManagement.autoUpdate": true,
"gopls": {
    "ui.semanticTokens": true
  },
```

### Describe the bug
The vscode-go extension does not install or look for `dlv` when running the "Go: Install/Update Tools" or "Go: Locate Configured Go Tools" command. See the output above under Version Information -  `dlv` isn't even mentioned. The attached screenshot shows that it's missing when trying to install the Go tools as well. 

Somehow the extension is still aware of `dlv`- when trying to debug for the first time, an error is shown and the extension offers to install `dlv`. So the bug is mainly a nuisance, since it requires me to keep delve up to date manually. 

### Steps to reproduce the behavior:
Not sure - this happens only on an M1 MacBook running macOS 12.4. I've reinstalled the extension, but the behavior doesn't change. On two Windows 11 PCs, the extension does offer to install/update `dlv` like expected, both in Windows as well as WSL. 

### Screenshots or recordings
If applicable, add screenshots or recordings to help explain your problem.
<img width="621" alt="Bildschirmfoto 2022-07-13 um 20 05 46" src="https://user-images.githubusercontent.com/1625465/178801189-8a5e58c0-ec10-4971-88fc-7f8853c09ccd.png">

**Comment by hyangah** [comment]
I suspect https://github.com/golang/vscode-go/blob/398f8ddce40cf576703e80a56b9ff99938de628a/src/goTools.ts#L161-166)

Thanks for the report. I am surprised that we didn't get this report earlier!

**Comment by jamalc** [comment]
This should be fixed in nightly and the [v0.35.0 RC](https://github.com/golang/vscode-go/releases/tag/v0.35.0-rc.3).
